docs/series/system/SystemSeriesA06.md

```diff
@@ -0,0 +1,94 @@
+# A06 ❯ Multimedia Codecs & Libraries for VFX Workstation Installation Guide
+<small>ℹ️ This article is part of AlmaLinux [System Series](/series/).</small>
+<hr>
+| 💡 | Experience Level  | ⭐⭐☆☆☆ |
+|--- | --------- | --------|
+| 📆 | <small>Last modified </small>| 2024-02-02
+| 🔧 | <small>Tested by <br> ↳ version \| platform \| date </small>| NOT TESTED YET |
+
+
+These packages have been needed for multiple software and we're either backported, requested or discovered has alternative for Alma Linux 9 from EL8.
```

Comment:
```suggestion
These packages have been needed for multiple software and were either backported, requested, or discovered as alternatives for AlmaLinux 9 for users migrating from enterprise Linux 9.
```

Reply:
These packages have been needed for multiple software, such has "Thinkbox Deadline" and were either backported, requested, or discovered as alternatives for AlmaLinux 9 for users migrating from enterprise Linux 8.